server/services/taskListService.ts

```diff
@@ -12,9 +12,19 @@ export default class TaskListService {
     this.formSections = Apply.sections
     this.taskStatuses = {}
 
+    let incompleteTasks = 0
+
     this.formSections.forEach(section => {
       section.tasks.forEach(task => {
-        this.taskStatuses[task.id] = getTaskStatus(task, application)
+        if (task.id === 'check-your-answers' && incompleteTasks) {
```

Comment:
sorry to be a pain but could we take the 'cannot start' logic out? It will be so much easier to manually test if we don't have to have all the tasks completed

Reply:
yes, have done